src/plasma/plasma_common.h

```diff
@@ -0,0 +1,41 @@
+#ifndef PLASMA_COMMON_H
+#define PLASMA_COMMON_H
+
+#include <cstring>
+#include <string>
+
+#include "logging.h"
+#include "status.h"
+
+constexpr int64_t kUniqueIDSize = 20;
+
+class UniqueID {
+ public:
+  static UniqueID from_random();
+  static UniqueID from_binary(const std::string& binary);
+  bool operator==(const UniqueID& rhs) const;
+  const uint8_t *data() const;
+  uint8_t *mutable_data();
+  std::string binary() const;
+  std::string sha1() const;
```

Comment:
What about a more generic name like `hash` or `hex`? Some of these IDs are just generated randomly and it's possible we'd change the hash function later.